Lib/gftools/builder/__init__.py

```diff
@@ -146,7 +146,12 @@ def fill_config_defaults(self):
                 self.config["familyName"] = designspace.sources[0].familyName
             else:
                 self.load_masters()
-                familynames = set([x.info.familyName for x in self.masters.values()])
+
+                familynames = set()
```

Comment:
More than happy to make this into a list comp but I've found list comps containing 2 loops are difficult to debug.